docs/generator/checklinks.sh

```diff
@@ -55,7 +55,7 @@ testinternal () {
 	ilnk=${3}
 	header=${ilnk//-/}
 	dbg "   - Searching for \"$header\" in $ifile"
-	tr -d '[],_.:? `'< "$ifile" | sed -e 's/-//g' -e "s/'//g" | grep -i "^\\#*$header\$" >/dev/null
+	tr -d '[],_.:? `'< "$ifile" | sed -e 's/-//g' -e "s/'//g" -e "s/(//g" -e "s/)//g" | grep -i "^\\#*$header\$" >/dev/null
```

Comment:
You could use `[]`, it means `any of`

> sed -e "s/[-'\)\(]//g"

Reply:
Not sure if it looks better 😄 Just saying.

Reply:
> tr -d '[],_.:? `'< "$ifile" | sed ...

Should we use only `sed` or `tr` there? i see we use both to achieve same goal - remove set of characters.

Reply:
Yes, I think I was having trouble putting a single quote inside a single quote and dash meant range. No need to be sticklers about it, it will all go away with the learn site.